src/pages/index.js

```diff
@@ -143,77 +139,121 @@ const editProfilePopup = new PopupWithForm(
 );
 editProfilePopup.setEventListeners();
 
+const avatarPopup = new PopupWithForm(
+  "#avatar-edit-modal",
+  handleAvatarFormSubmit
+);
+avatarPopup.setEventListeners();
+
 // ---------------- Form Handlers ----------------
 function handleProfileFormSubmit(values) {
   console.log(values);
   userInfo.setUserInfo({
     title: values.title,
     description: values.description,
   });
-  /* closeModal(profileEditModal); */
-  // closeModal(editProfilePopup);
   editProfilePopup.close();
 }
 
+function handleAvatarFormSubmit(values) {
```

Comment:
Every server request in `index.js` should look something like this:

```js
api.someMethod()
.then(res => {
// here is the response that should be used to update the DOM.
// also in `then` you should close the popup
})
.catch(err => // here you catch possible errors)
.finally(() => // here you return the default button text back )
``` 

Please, correct it in the whole code